src/Controller/EnMarche/MembershipController.php

```diff
@@ -146,6 +212,11 @@ public function activateAction(Adherent $adherent, AdherentActivationToken $acti
 
         try {
             $this->get('app.adherent_account_activation_handler')->handle($adherent, $activationToken);
+            $this->addFlash('info', $this->get('translator')->trans('adherent.activation.success'));
```

Comment:
We gonna have a useless flash message in session if the guy is redirected to an OAuth app. That's why the query string `from_activation` is used instead of a flash message.